--- assimp/code/CMakeLists.txt.orig	2025-02-11 01:18:39.000000000 +0100
+++ assimp/code/CMakeLists.txt	2025-02-11 14:05:34.622797702 +0100
@@ -1435,7 +1435,7 @@
     target_link_libraries(assimp PRIVATE ${draco_LIBRARIES})
   endif()
 ELSE()
-  TARGET_LINK_LIBRARIES(assimp ${ZLIB_LIBRARIES} ${OPENDDL_PARSER_LIBRARIES})
+  TARGET_LINK_LIBRARIES(assimp z ${OPENDDL_PARSER_LIBRARIES})
   if (ASSIMP_BUILD_DRACO)
     target_link_libraries(assimp ${draco_LIBRARIES})
   endif()
